decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -34,6 +34,10 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         registerObserverForKeyboardNotification()
         self.hideKeyboardWhenTappedAround()
         
+        //set border for dropdown list
```

Comment:
there should be a single whitespace between the last slash and the first word in the comment
`// set ...`